config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```

Comment:
1. Create new file `database.yml.example` in folder `config/` with the same content <- this file should be push on github
2. gitignore `config/database.yml` file 
3. Configuring MySQL DB instead of the SQLite3
refs: https://edgeguides.rubyonrails.org/configuring.html#configuring-a-mysql-or-mariadb-database
sample file: https://gist.github.com/mmizutani/5a2d84cc438675d3bcbfd9cc59de53ba
For the 3rd steps:
- Replace `database, username, password` corresponding your database in setting file
- Run `rails db:create` to create your DB